### PR TITLE
Add toml python extra to pull in a toml parser

### PR DESCRIPTION
**Pull request by gotmax23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ It can be installed with pip:
 
     pip install antsibull-changelog
 
+For python projects, `antsibull-changelog release` can retrieve the current
+version from `pyproject.toml`.
+You can install the project with
+
+    pip install antsibull-changelog[toml]
+
+to pull in the necessary toml parser for this feature.
```
Comment (felixfontein) [inline README.md:36]: How about mentioning that this is only needed for Python < 3.11?
```suggestion
to pull in the necessary toml parser (for Python < 3.11) for this feature.
```
Besides that, LGTM.

Comment (gotmax23) [reply README.md:36]: Good point! https://github.com/ansible-community/antsibull-changelog/pull/126/commits/80ec0c96fe0a618e97706289a27df3c80422122f.

**Comment by felixfontein** [comment]
@gotmax23 thanks!

**Comment by gotmax23** [comment]
Thanks @felixfontein for the review!

**Pull request merged**